controllers/operands/kubevirt.go

```diff
@@ -853,6 +864,10 @@ func getFeatureGateChecks(featureGates *hcov1beta1.HyperConvergedFeatureGates) [
 		fgs = append(fgs, kvAlignCPUs)
 	}
 
+	if featureGates.PrimaryUserDefinedNetworkBinding != nil && *featureGates.PrimaryUserDefinedNetworkBinding {
+		fgs = append(fgs, kvDynamicPodInterfaceNaming)
```

Comment:
@EdDev @orelmisan 
it enabled the FG only if HCO FG `PrimaryUserDefinedNetworkBinding` is enabled

Reply:
And this is exactly what I commented on in the other [thread](https://github.com/kubevirt/hyperconverged-cluster-operator/pull/3139#discussion_r1836070837).

Such a dependency is IMO wrong.

Reply:
> If HCO is enabling a FG, it means it has guarantees it will never be removed, which is hard to commit to for Alpha or Beta features.

But it is enabled only if HCO optional FG `PrimaryUserDefinedNetworkBinding` is enabled,
which is also a FG that has graduation, doesn't it change the picture?

anyhow please decide if you want me to remove it,
it means that the user will need to patch the HCO with a json patch, which affects the kubevirt CR.

@maiqueb @qinqon 
fyi please

Reply:
I already voted and explained my vote in detail. You are welcome to challenge my reasoning.

Reply:
so what do you suggest @EdDev ?

Reply:
> It needs to graduate to GA on Kubevirt and only then be permanently added to HCO as enabled. And it should not be dependent on any other feature lifecycle.

@maiqueb  this ^^

Reply:
@oshoval I tend to agree here ...

Let's let @EdDev and his team handle this. 

Let us know when the feature is GA'd in KubeVirt. Until then, I guess, nothing to do.